### PR TITLE
Use Github Actions

### LINKED ISSUE #336: Add github actions for testing (instead of Travis)
Github actions is native to github so, ideally, we should use it

### PR DESCRIPTION
**Pull request by genevieveluyt**
##### Description

Use Github Actions for CI

Fixes #336 

##### Type of change

- [ ] Bug fix :bug:
- [ ] New feature :sparkles:
- [ ] This change requires a documentation update :book:
- [ ] Breaking changes :warning:
##### How Has This Been Tested?

- [ ] Test A
- [ ] Test B

##### Checklist:

- [ ] I have :tophat: my changes (A 🎩 specifically includes pulling down changes, setting them up, and manually testing the changed features and potential side effects to make sure nothing is broken)
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] The test coverage did not decrease
- [ ] I have signed the appropriate [Contributor License Agreement](https://cla.shopify.com/)

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: CI
+on:
+  push:
```
Comment (genevieveluyt) [inline .github/workflows/ci.yml:3]: Will this still re-run CI if you push commits to a PR? For example, if tests fail and you push a new commit to fix the test, I think we want CI to run again?

Comment (dani-santos-code) [reply .github/workflows/ci.yml:3]: Every time we push a commit to this PR, ci will re-run. You can see the little check next to every commit:

Screen Shot 2021-05-25 at 4 46 34 PM

Is that what you mean? 🤔

Comment (genevieveluyt) [reply .github/workflows/ci.yml:3]: Nice yea!

**Comment by genevieveluyt** [comment]
Looks good! I can't approve because I opened the PR but 👍

**Review by dani-santos-code — APPROVED** [review]

**Pull request merged**